tests/fixtures/modules/import/error-strict.src.js

```diff
@@ -0,0 +1,5 @@
+import house from "house";
```

Comment:
I take that back, let's keep it here so we can get a different error if for some reason the test is not run for `sourceType: 'module'`